lib/controllers/ServiceBrokerApiController.js

```diff
@@ -212,19 +213,14 @@ class ServiceBrokerApiController extends BaseController {
   }
 
   validateRequest(req, res) {
+    /* jshint unused:false */
     if (req.instance.async && (_.get(req, 'query.accepts_incomplete', 'false') !== 'true')) {
-      res.status(CONST.HTTP_STATUS_CODE.UNPROCESSABLE_ENTITY).send({
-        error: 'AsyncRequired',
-        description: 'This request requires client support for asynchronous service operations.'
-      });
+      throw new UnprocessableEntity('This request requires client support for asynchronous service operations.', 'AsyncRequired');
     }
     const operationType = _.get(req, 'operation_type');
     if (_.includes([CONST.OPERATION_TYPE.CREATE], operationType) &&
       (!_.get(req.body, 'space_guid') || !_.get(req.body, 'organization_guid'))) {
-      res.status(CONST.HTTP_STATUS_CODE.BAD_REQUEST).send({
-        error: 'BadRequest',
-        description: 'This request is missing mandatory organization guid and/or space guid.'
-      });
+      throw new BadRequest('This request is missing mandatory organization guid and/or space guid.');
```

Comment:
do you want to put 'BadRequest' as second parameter just like L218?

Reply:
Hi @sauravmndl,

For Bad Request, there is no definite answer to what should be the error, and also OSB checker only checks for the error code 400. Hence, let's keep it as is.